wrapper/src/test/java/integration/container/aurora/mysql/mariadbdriver/AuroraMysqlAwsIamIntegrationTest.java

```diff
@@ -134,4 +132,12 @@ void test_AwsIam_UserInConnStr() throws SQLException {
                 awsIamProp)
     );
   }
+
+  private Properties initAwsIamPropsMariaDB(final String user, final String password) {
+    final Properties props = initAwsIamProps(user, password);
+    props.setProperty("sslMode", "verify-ca");
+    props.setProperty("serverSslCert", "/app/test/resources/rds-ca-2019-root.pem");
```

Comment:
I thought that is not needed. Does it?

Reply:
It is required for MariaDB, even with an instance endpoint.